packages/jqr/install

```diff
@@ -5,10 +5,10 @@ set -e
 OS_DISTRIBUTION=$(lsb_release -cs)
 
 apt-get update -qq
-apt-get install -y software-properties-common python-software-properties
 
-# these sysdeps are available by default in bionic
-if [ "${OS_DISTRIBUTION}" == "xenial" ] || [ "${OS_DISTRIBUTION}" == "trusty" ]; then
+# this sysdep is available by default in bionic
+if [[ "${OS_DISTRIBUTION}" == "xenial" ]]; then
+  apt-get install -y software-properties-common python-software-properties
```

Comment:
We do not need these on Bionic? Are these just necessary for hitting the PPA?

Reply:
They are needed for `add-apt-repository`. If we aren't using it, we don't need 'em